docs/custom-capo.md

```diff
@@ -0,0 +1,58 @@
+# Creating a custom cluster api provider ovirt and adding it to the cluster
+
+When we develop the CAPO its usually helpful to test it against a running cluster.
+The following doc explains how to do it.
+
+## Replacing the docker image
+
+1.Lunch a cluster:
+
+Start an Openshift cluster on top of oVirt as usual with the required release image.
+
+2. Create the docker image:
+
+** replace ${username} and ${tag} with the correct values
```

Comment:
** doesn't show special char in Markdown. 
I would move this comment below the podman commands below, like:

**NOTE**:
${username} and ${tag} make be replaced with the the proper quay.io username and cluster-api-provider-ovirt tag.